### PR TITLE
feat: Covid Appreciation Letter

### PR DESCRIPTION
**Pull request by SiaChenRen**

### CODE REVIEW

File: src/templates/covidAppreciationLetter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { CovidCOATemplate } from "./template";
+
+export const covidCOATemplates = [
```
Comment (Nebulis) [inline src/templates/covidAppreciationLetter/index.tsx:3]: let's avoid acronym, change coa to what it's supposed to mean

also the if is Letter Of Appreciaation, so should have been loa right ? :)

File: src/templates/covidAppreciationLetter/types.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { v2 } from "@govtechsg/decentralized-renderer-react-components";
+
+export interface CovidCOAInterface extends v2.OpenAttestationDocument {
+  name: string;
```
Comment (Nebulis) [inline src/templates/covidAppreciationLetter/types.tsx:4]: what's this ?

File: src/templates/covidAppreciationLetter/template.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  }
+
+  .template-footer p span {
+    margin-right: 5px;
+  }
+`;
+export const CovidCOATemplate: FunctionComponent<TemplateProps<CovidCOAInterface> & { className?: string }> = ({
+  document
+}) => {
+  const recipientName = get(document, "recipient.name");
```
Comment (Nebulis) [inline src/templates/covidAppreciationLetter/template.tsx:84]: forbidden, just use plain . or ?.

File: src/templates/covidAppreciationLetter/template.tsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  const recipientFirstName = get(document, "recipient.firstName");
+  const recipientDivision = get(document, "recipient.division");
+  const recipientSite = get(document, "recipient.site");
+  const signature = get(document, "signature");
```
Comment (Nebulis) [inline src/templates/covidAppreciationLetter/template.tsx:88]: something is wrong, according the thetype, it should be signatory.signature

File: src/templates/covidAppreciationLetter/types.tsx (hunk 2 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/types.tsx:4]: The data schema for the letter of appreciation

File: src/templates/covidAppreciationLetter/types.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  recipient: {
+    name: string;
+    firstName: string;
+    division: string;
+    site: string;
+  };
+  signatory: {
+    name: string;
```
Comment (Nebulis) [inline src/templates/covidAppreciationLetter/types.tsx:12]: what's name / position / organisation for ? it's not used in the template

File: src/templates/covidAppreciationLetter/index.tsx (hunk 1 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/index.tsx:3]: Done

File: src/templates/covidAppreciationLetter/types.tsx (hunk 2 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/types.tsx:4]: Done

File: src/templates/covidAppreciationLetter/template.tsx (hunk 3 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/template.tsx:84]: Done

File: src/templates/covidAppreciationLetter/template.tsx (hunk 4 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/template.tsx:88]: Done

File: src/templates/covidAppreciationLetter/types.tsx (hunk 5 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/types.tsx:12]: Done

File: src/templates/covidAppreciationLetter/index.tsx (hunk 1 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/index.tsx]: Done

File: src/templates/covidAppreciationLetter/types.tsx (hunk 2 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/types.tsx]: Done

File: src/templates/covidAppreciationLetter/template.tsx (hunk 3 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/template.tsx]: Done

File: src/templates/covidAppreciationLetter/template.tsx (hunk 4 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/template.tsx]: Done

File: src/templates/covidAppreciationLetter/types.tsx (hunk 5 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/types.tsx]: Done

**Review by Nebulis — APPROVED** [review]

**Pull request merged**